src/test/java/org/opensearch/knn/index/IndexUtilTests.java

```diff
@@ -67,4 +69,26 @@ public void testGetLoadParameters() {
         assertEquals(spaceType2.getValue(), loadParameters.get(SPACE_TYPE));
         assertEquals(efSearchValue, loadParameters.get(HNSW_ALGO_EF_SEARCH));
     }
+
+    public void testGetFieldMappingNonNestedField() {
+        Map<String, Object> properties = new HashMap<>();
+        properties.put("top_level_field", "top_level_value");
```

Comment:
nit: you can use `Map.of()` to initialize map with data

Reply:
should we also add a case with empty map?

Reply:
Thanks, updated to use `Map.of()` and added empty map test case.